src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_AssignmentOperator.cs

```diff
@@ -264,6 +277,17 @@ private BoundExpression MakeStaticAssignmentOperator(
             }
         }
 
+        private bool IsPropertyWithByValPossiblyStructReceiverWhichHasLocationAndCanChangeValueBetweenReads(BoundExpression rewrittenReceiver, PropertySymbol property)
```

Comment:
>IsPropertyWithByValPossiblyStructReceiverWhichHasLocationAndCanChangeValueBetweenReads [](http://example.com/codeflow?start=21&length=86)

Consider renaming to `IsExtensionPropertyWithByValPossiblyStructReceiverWhichHasHomeAndCanChangeValueBetweenReads` as only **extension** properties qualify and to avoid multiplying concepts ("home" versus "location").